examples/getting_started/flower/README.md

```diff
@@ -0,0 +1,22 @@
+# Flower App (PyTorch) in NVIDIA FLARE
+
+In this example, we run 2 Flower clients and Flower Server in parallel using NVFlare's simulator with the JobAPI.
+
+## Preconditions
+
+To run Flower code in NVFlare, simply install the requirements and run the below script. 
+
+Note, this code is directly copied from Flower's [app-pytorch](https://github.com/adap/flower/tree/main/examples/app-pytorch) example.
+
+## Install dependencies
+To run this job with NVFlare, we first need to install the dependencies.
+```bash
+pip install -r requirements.txt
```

Comment:
we might need to ping specific version here as Flower is still changing their api implementation?

Reply:
Yes, we do need to pin a specific version of flower.
I asked them for a fix version number but have not got response from them. Their main developer Pan is attending a conference. Not sure when he will be able to get back to us.